examples/mesos/README.md

```diff
@@ -0,0 +1,14 @@
+# Launching Weave Scope on Apache Mesos Cluster
```

Comment:
Are you suggesting adding `minimesos` to `site/installing.md` as a platform along with k8, DC/OS and Amazon?